packages/patternfly-4/react-table/src/components/Table/Body.js

```diff
@@ -20,35 +21,69 @@ const defaultProps = {
 
 class ContextBody extends React.Component {
   onRow = (row, props) => {
-    const { onRowClick } = this.props;
+    const { onRowClick, editConfig } = this.props;
     return ({
-      isExpanded: row.isExpanded,
-      isOpen: row.isOpen,
-      onClick: (event) => onRowClick(event, row, props)
+      row,
+      rowProps: props,
+      onMouseDown: (event) => {
```

Comment:
this seems like kind of a hack, but works. I wonder if proposing an upstream PR to reactabular in parallel to allow hooks into `onChange`\`onBlur` would be a good idea?

Reply:
Where would the hooks be? I am not sure something like this would help. 
The idea of this is to get ahead of cell onBlur callbacks. Because of the cell's onBlur callback, the cell will be scrapped and then constructed again. And then we get a chance to fire callbacks (captured here) on our new instance.

Reply:
it is a hack nevertheless :)